src/coreclr/clrfeatures.cmake

```diff
@@ -39,3 +39,7 @@ endif()
 if (CLR_CMAKE_TARGET_OSX OR CLR_CMAKE_TARGET_MACCATALYST OR CLR_CMAKE_TARGET_IOS OR CLR_CMAKE_TARGET_TVOS)
   set(FEATURE_OBJCMARSHAL 1)
 endif()
+
+if(NOT DEFINED FEATURE_STRESSLOG AND NOT CLR_CMAKE_TARGET_ALPINE_LINUX AND (CLR_CMAKE_TARGET_ARCH_AMD64 OR CLR_CMAKE_TARGET_ARCH_ARM64))
+  option(FEATURE_STRESSLOG ON)
```

Comment:
Consider naming this flag `FEATURE_STRESSLOG_ANALYZER`. StressLog support is always enabled in the product, it is the analyzer that we want to turn on or off.